examples/meta/src/converter/factor_analysis.sg.in

```diff
@@ -0,0 +1,17 @@
+File f_feats_train = read_csv("@SHOGUN_DATA@/fm_train_real.dat")
+
+#![create_features]
+Features feats = create_features(f_feats_train)
+#![create_features]
+
+#![set_parameters]
+Transformer fa = create_transformer("FactorAnalysis", target_dim=2, max_iteration=200, epsilon=0.00001)
```

Comment:
can print `fa` in octave and share the output. I suspect that epsilon value might not be passed properly.

Reply:
How should I do this? As in should I make the change in the generated octave file?

Reply:
Simplest would be to run the octave code in the octave shell, or yes, modify the octave script